keyboards/ibm/model_m/ctrl_m/readme.md

```diff
@@ -0,0 +1,30 @@
+# ctrl-M
+
+![ctrl-M PCB](https://github.com/nuess0r/ctrl-M/assets/14843228/2c2c5d48-6dd1-4914-bfdc-a574e429aa96)
```

Comment:
Could this be rehosted on Imgur or the like?

Reply:
done